src/components/Comment/Comment.tsx

```diff
@@ -231,7 +237,9 @@ export const Comment = ({
   isMuted,
   toggleMuteStatus
 }: Props) => {
-  const commentControlsButtonStyles = commentControlsButton(pillar);
+  const commentControlsButtonWrapperStyles = commentControlsButtonWrapper(
+    pillar
+  );
```

Comment:
Would it make things simpler to rename this to `buttonOverrides(pillar)` and use the function directly?